src/components/MembersSectionNew/components/MembersCard/membersCard.module.css

```diff
@@ -30,30 +22,74 @@
   height: 350px;
 }
 
-.member_card:hover {
-  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+.member_card__image {
+  visibility: hidden;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 250px;
+  border-top-left-radius: 12px;
+  border-top-right-radius: 12px;
 }
 
-.member_card__image_container {
-  margin: 0 auto;
-  margin-top: 4px;
-  position: relative;
-  border-radius: 50%;
-  width: 12rem;
-  height: 12rem;
+.member_card__image_hover {
+  position: absolute;
+  transition: 0.2s all ease-out;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 100%;
+  height: 250px;
+  border-top-left-radius: 12px;
+  border-top-right-radius: 12px;
+  top: 0;
 }
 
-.member_card__image {
-  border-radius: 50%;
+.member_card__info {
+  z-index: 2;
+  background-color: #fff;
+  border-bottom-left-radius: 12px;
+  border-bottom-right-radius: 12px;
+  padding: 8px 24px 10px 24px;
+  text-align: center;
 }
 
 .member_card__username {
   font-size: 22px;
   font-weight: 600;
 }
 
+.member_card__designation {
+  font-weight: 600;
+}
+
 .member_card__socials {
-  gap: 2.5;
+  justify-content: center;
   margin-top: 3px;
-  margin-bottom: 3px;
-}
\ No newline at end of file
+  margin-bottom: 14px;
+}
+
+.member_card:hover {
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+
+.member_card:hover {
+  box-shadow: 0px 30px 18px -8px rgba(0, 0, 0, 0.1);
+  transform: scale(1.1, 1.1);
+}
+
```

Comment:
here as well

Reply:
box-shadow should be fixed so using rem instead px is not an good idea